mappings/net/minecraft/client/gui/widget/PressableWidget.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_4264 net/minecraft/client/gui/widget/PressableWidget
+	COMMENT A pressable widget can be "pressed" when it is clicked or when enter or space
+	COMMENT keys are pressed when it is selected.
```

Comment:
It's pain to read a sentence which contains three "when" imo.
```suggestion
	COMMENT A pressable widget can be "pressed" when it is clicked. It can also be pressed
	COMMENT when enter or space keys are pressed when it is selected.
```

Reply:
I will reword this locally fyi, saying this has a "press" action in general